docs/source/deploy_rhel7_centos7.rst

```diff
@@ -1,149 +1,213 @@
 Deploying on RHEL 7 / CentOS 7 / Amazon Linux 2
 ===============================================
 
-Quickstart
-----------
+*Instructions applicable for RHEL 7, CentOS 7 and Amazon Linux 2. Tested
+on a*
+`CentOS 7.8.2003 Docker image <https://hub.docker.com/_/centos>`__
+*of 2020-05-04.*
 
-.. container:: content-tabs
+Installation on RHEL/CentOS 7
+-----------------------------
 
-::
-
-   .. tab-container:: rhel7
-       :title: RHEL/CentOS 7
+GovReady-Q requires Python 3.6 or higher and several Linux packages to
+provide full functionality. Execute the following commands:
 
-       .. rubric:: Installing on RHEL/CentOS 7
-       
-       *Instructions  applicable RHEL 7, CentOS 7 and Amazon Linux 2.*
+.. code:: bash
 
-       GovReady-Q calls requires Python 3.6 or higher to run and several Linux packages to provide full functionality.
+   # enable IUS repository
+   sudo yum install https://centos7.iuscommunity.org/ius-release.rpm
+   sudo yum update
 
-       .. code-block:: bash
+   # install dependencies
+   sudo yum install \
+   python36u python36u-pip \
+   unzip git2u jq nmap-ncat \
```

Comment:
@peterkaminski Do we need `nmap-ncat` or ist a holdover from continuous ATO?

Reply:
it's a holdover, but it's also in Dockerfile.  i figured we should remove them together.